test/Dialect/ESI/CMakeLists.txt

```diff
@@ -10,9 +10,9 @@ set(LLVM_LINK_COMPONENTS
   Support
   )
 
-add_llvm_tool(esi-tester
- esi-tester.cpp
-)
+add_circt_tool(esi-tester
+  esi-tester.cpp
+  )
```

Comment:
Standardizing formatting is fine here, but probably stick to the convention where closing paren is on column 0? More importantly, this is not done consistently in this PR so let's pick one or the other :+1: .

Reply:
Great, no, you're right.  Thank you!